### PR TITLE
*: add community governace and sig governace, in Chinese

### PR DESCRIPTION
**Pull request by winkyao**
Signed-off-by: winkyao <golangwink@gmail.com>

In Chinese, It will be translated into English later.

@gingerkidney

### CODE REVIEW

File: GOVERNACE-zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+![community organization](media/governace/community_organization.png)
+
+社区架构分成两个部分，第一部分是社区角色架构图，第二部分为社区小组内部的角色分工，以及社区角色的晋升路径。
+
+PMC：项目管理委员会，为项目核心管理团队，参与 roadmap 制定与社区相关的重大决议。
+
+Developer Group：
+
+* Maintainer：项目的规划和设计者，拥有合并主干分支的权限，从 Committer 中产生。
+* Committer：由 maintainer 或 PMC 推荐，对 TiDB 有突出贡献的 contributor。需要独立完成至少一个 feature 或修复重大 bug。
```
Comment (lonng) [inline GOVERNACE-zh_CN.md]:
```suggestion
* Committer：由 maintainer 或 PMC 推荐，对 TiDB 有突出贡献的 contributor，需要独立完成至少一个 feature 或修复重大 bug。
```

File: GOVERNACE-zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* Reviewer：进阶 contributor，负责 review 社区代码，对代码有 LGTM 权限。可以参与到后续项目的设计中。
```
Comment (lonng) [inline GOVERNACE-zh_CN.md]:
```suggestion
* Reviewer：进阶 contributor，负责 review 社区代码，对代码有 LGTM 权限，可以参与到后续项目的设计中。
```

File: GOVERNACE-zh_CN.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* Active Contributor：活跃的社区贡献者，有两位 reviewer 以上的社区角色提名通过。可以参与到 SIG 中。至少合并 8 个 pr 以上。
```
Comment (lonng) [inline GOVERNACE-zh_CN.md]:
```suggestion
* Active Contributor：活跃的社区贡献者，有两位 reviewer 以上的社区角色提名通过，可以参与到 SIG 中。至少合并 8 个 PR 以上。
```

File: GOVERNACE-zh_CN.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+![developer group](media/governace/developer_group.png)
+
+社区对外的小组有 SIG 与 Working group（WG）。第二部分组织架构图对 SIG 和 WG 的定义、小组内部角色和个角色晋升通道进行了说明。
+
+### 组织：
+
+* SIG：专项兴趣小组，目前为邀请制。成员为在某一专项活跃的 contributor。小组最终目的为培养社区人才，帮助大家晋升为更高阶的 reviewer，committer 以及 maintainer。
+
+* WG：开放的工作组，社区成员可以无门槛加入。工作组是为了完成一个短期项目而建立的临时小组，项目完成即解散。一个工作组可以横跨多个兴趣小组。经过一段时间的培训后优秀的社区小伙伴会被选入某个兴趣小组进行进一步深造。
+
```
Comment (lonng) [inline GOVERNACE-zh_CN.md]:
```suggestion
```

File: GOVERNACE-zh_CN.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+### 内部角色：
+* Tech Leader：小组技术负责人，对整个小组的技术走向进行把控。负责制定学习计划、成员考核制度等。一个小组可以有多个 Tech Leader。
```
Comment (lonng) [inline GOVERNACE-zh_CN.md]: Add an blank line above this line please.

File: GOVERNACE-zh_CN.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+* 其余角色见社区组织架构图
+
+## 晋升通道：
+1. Active Contributor -> Reviewer
```
Comment (lonng) [inline GOVERNACE-zh_CN.md]: ditto

File: GOVERNACE-zh_CN.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+PMC：项目管理委员会，为项目核心管理团队，参与 roadmap 制定与社区相关的重大决议。
+
+Developer Group：
+
+* Maintainer：项目的规划和设计者，拥有合并主干分支的权限，从 Committer 中产生。
+* Committer：由 maintainer 或 PMC 推荐，对 TiDB 有突出贡献的 contributor。需要独立完成至少一个 feature 或修复重大 bug。
+* Reviewer：进阶 contributor，负责 review 社区代码，对代码有 LGTM 权限。可以参与到后续项目的设计中。
+* Active Contributor：活跃的社区贡献者，有两位 reviewer 以上的社区角色提名通过。可以参与到 SIG 中。至少合并 8 个 pr 以上。
+* Contributor：社区贡献者，在 TiDB 核心 repo 有一个以上被合并的 PR。
```
Comment (lonng) [inline GOVERNACE-zh_CN.md]:
```suggestion
* Contributor：社区贡献者，在 TiKV 核心 repo 有一个以上被合并的 PR。
```

File: community-membership-zh_CN.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
+# 社区角色
+
+## Maintainer
+
+项目的规划和设计者，拥有合并主干分支的权限，从 Committer 中产生。
+他们必须对子项目的健康表现出良好的判断力和责任感。维护者必须直接或通过委派这些职责来设置技术方向并为子项目做出或批准设计决策。
+
+### 如何成为
+
+* 对 TiDB 的技术目标和方向有非常深入的了解
```
Comment (lonng) [inline community-membership-zh_CN.md]:
```suggestion
* 对 TiKV 的技术目标和方向有非常深入的了解
```

File: community-membership-zh_CN.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+
+* 制定子项目的技术设计和决策。
+* 设置子项目的技术方向和优先级。
+* 参与版本发布，规划项目发展路线
+* 指导 contributor，reviewer 和 committer 对项目持续贡献
+* 保证子项目的整体质量
+
+<h2 id="committer">Committer</h2>
+
+由 maintainer 或 PMC 推荐，是对 TiDB 有突出贡献的 contributor。需要独立完成至少一个 feature 或修复重大 bug。
```
Comment (lonng) [inline community-membership-zh_CN.md]:
```suggestion
由 maintainer 或 PMC 推荐，是对 TiKV 有突出贡献的 contributor。需要独立完成至少一个 feature 或修复重大 bug。
```

File: community-membership-zh_CN.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+
+### 如何成为
+
+* 表现出良好的技术判断力
+* 至少是两个以上子项目的 reviewer
+* 审阅或合并了一定数量的 TiDB 代码
```
Comment (lonng) [inline community-membership-zh_CN.md]:
```suggestion
* 审阅或合并了一定数量的 TiKV 代码
```

File: community-membership-zh_CN.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+* 对 TiDB 性能提升做出突出贡献；修复了重大 bug
```
Comment (lonng) [inline community-membership-zh_CN.md]:
```suggestion
* 对 TiKV 性能提升做出突出贡献；修复了重大 bug
```

File: community-membership-zh_CN.md [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+* 由两名或以上在任 Maintainer 提名通过
+
+### 责任与义务
+
+* 整体把控项目的代码质量 
+* 指导 contributor 和 reviewer 对 TiDB 持续贡献
```
Comment (lonng) [inline community-membership-zh_CN.md]:
```suggestion
* 指导 contributor 和 reviewer 对 TiKV 持续贡献
```

File: community-membership-zh_CN.md [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+* 定期参与 Committer 会议，对项目发展有优先知情权
+
+<h2 id="reviewer">Reviewer</h2>
+
+负责审查社区提交的代码，保证代码的质量和正确性，从 contributor 中产生。
+
+### 如何成为
+
+* 熟悉代码库
+* 为 TiDB 子模块贡献一定数量的代码
```
Comment (lonng) [inline community-membership-zh_CN.md]:
```suggestion
* 为 TiKV 子模块贡献一定数量的代码
```

File: community-membership-zh_CN.md [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+* 及时响应社区提交的 PR
+* 定期参加专项兴趣小组（SIG）的研讨会
+* 对代码拥有 LGTM 的权限
+
+## Active Contributor
+
+活跃的社区贡献者。
+
+### 如何成为
+* 一年内对 TiDB 贡献 8 个 pr 及以上
```
Comment (lonng) [inline community-membership-zh_CN.md]:
```suggestion
* 一年内对 TiKV 贡献 8 个 pr 及以上
```

File: community-membership-zh_CN.md [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+* 得到两位现任 reviewer  的认可
+
+### 责任与义务
+* 自动获得进入专项兴趣小组（SIG）的资格
+
+## Contributor
+
+* 社区贡献者，在 TiDB 核心 repo 有一个或以上被合并的 PR。
```
Comment (lonng) [inline community-membership-zh_CN.md]:
```suggestion
* 社区贡献者，在 TiKV 核心 repo 有一个或以上被合并的 PR。
```

File: sig-governace/SIG-GOVERNACE-zh_CN.md [hunk 16]
```diff
@@ -0,0 +1,75 @@
+# SIG Governance
+SIG 全称 Special Interest Group。SIG 在社区架构中的位置请见：社区架构&角色。SIG 主要聚集一批 Active Contributor，对某一个或多个 TiDB 的模块深入研究 & 贡献，在 SIG 内部一步步晋升为 Reviewer，Committer。
```
Comment (lonng) [inline sig-governace/SIG-GOVERNACE-zh_CN.md]:
```suggestion
SIG 全称 Special Interest Group。SIG 在社区架构中的位置请见：社区架构 & 角色。SIG 主要聚集一批 Active Contributor，对某一个或多个 TiDB 的模块深入研究 & 贡献，在 SIG 内部一步步晋升为 Reviewer，Committer。 
```

**Comment by Hoverbear** [comment]
@winkyao This is a major organizational change that must be accepted by our maintainers through our defined RFC process. https://github.com/tikv/rfcs/

**Comment by winkyao** [comment]
@Hoverbear Of course, I already asked @dcalvin helping to finish this progress.

**Comment by Hoverbear** [comment]
Ok great. :) Looking forward to seeing the RFC. In the meantime this PR should not be merged.

**Comment by winkyao** [comment]
The diagrams in this PR should be redesigned to meet the tikv community. @gingerkidney

**Comment by winkyao** [comment]
RFC for TiKV Community Reorg: https://github.com/tikv/rfcs/pull/29

File: GOVERNACE-zh_CN.md [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+![community organization](media/governace/community_organization.png)
+
+社区架构分成两个部分，第一部分是社区角色架构图，第二部分为社区小组内部的角色分工，以及社区角色的晋升路径。
+
+PMC：项目管理委员会，为项目核心管理团队，参与 roadmap 制定与社区相关的重大决议。
+
+Developer Group：
+
+* Maintainer：项目的规划和设计者，拥有合并主干分支的权限，从 Committer 中产生。
+* Committer：由 maintainer 或 PMC 推荐，对 TiKV 有突出贡献的 contributor，需要独立完成至少一个 feature 或修复重大 bug。
```
Comment (gingerkidney) [inline GOVERNACE-zh_CN.md]:
```suggestion
* Committer：由 Maintainer 或 PMC 推荐，对 TiKV 有突出贡献的 Contributor，需要独立完成至少一个 feature 或修复重大 bug。
```

File: GOVERNACE-zh_CN.md [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+* Reviewer：进阶 contributor，负责 review 社区代码，对代码有 LGTM 权限，可以参与到后续项目的设计中。
```
Comment (gingerkidney) [inline GOVERNACE-zh_CN.md]:
```suggestion
* Reviewer：进阶 Contributor，负责 review 社区代码，对代码有 LGTM 权限，可以参与到后续项目的设计中。
```

File: GOVERNACE-zh_CN.md [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+* Active Contributor：活跃的社区贡献者，有两位 reviewer 以上的社区角色提名通过。可以参与到 SIG 中，至少合并 8 个 PR 以上。
```
Comment (gingerkidney) [inline GOVERNACE-zh_CN.md]:
```suggestion
* Active Contributor：活跃的社区贡献者，有两位 Reviewer 以上的社区角色提名通过。可以参与到 SIG 中，至少合并 8 个 PR 以上。
```

File: GOVERNACE-zh_CN.md [hunk 20]
```diff
@@ -0,0 +1,51 @@
...
+* Contributor：社区贡献者，在 TiKV 核心 repo 有一个以上被合并的 PR。
+
+Organizer Committee：项目组织委员会，负责执行、输出，推广技术内容的组织者们，成员包括各地区用户组组长以及社区活动负责人。
+
+## 开发者社区组织架构
+
+![developer group](media/governace/developer_group.png)
+
+社区对外的小组有 SIG 与 Working group（WG）。第二部分组织架构图对 SIG 和 WG 的定义、小组内部角色和个角色晋升通道进行了说明。
```
Comment (gingerkidney) [inline GOVERNACE-zh_CN.md]:
```suggestion
社区对外的小组有 Special Interest Group SIG 与 Working Group（WG）。第二部分组织架构图对 SIG 和 WG 的定义、小组内部角色和个角色晋升通道进行了说明。
```

File: GOVERNACE-zh_CN.md [hunk 21]
```diff
@@ -0,0 +1,51 @@
...
+
+### 组织：
+
+* SIG：专项兴趣小组，目前为邀请制。成员为在某一专项活跃的 contributor。小组最终目的为培养社区人才，帮助大家晋升为更高阶的 reviewer，committer 以及 maintainer。
```
Comment (gingerkidney) [inline GOVERNACE-zh_CN.md]:
```suggestion
* SIG：专项兴趣小组，目前为邀请制。成员为在某一专项活跃的 Contributor。小组最终目的为培养社区人才，帮助大家晋升为更高阶的 Reviewer，Committer 以及 Maintainer。
```

File: community-membership-zh_CN.md [hunk 22]
```diff
@@ -0,0 +1,72 @@
...
+* 积极组织技术研讨会，并提出有建设性的专案
+* 能处理代码实现过程中的复杂问题
+* PMC 内的成员投票一致通过
+
+### 责任与义务
+
+* 制定子项目的技术设计和决策。
+* 设置子项目的技术方向和优先级。
+* 参与版本发布，规划项目发展路线
+* 指导 contributor，reviewer 和 committer 对项目持续贡献
```
Comment (gingerkidney) [inline community-membership-zh_CN.md]:
```suggestion
* 指导 Contributor，Reviewer 和 Committer 对项目持续贡献
```

File: community-membership-zh_CN.md [hunk 23]
```diff
@@ -0,0 +1,72 @@
...
+* 保证子项目的整体质量
+
+<h2 id="committer">Committer</h2>
+
+由 maintainer 或 PMC 推荐，是对 TiKV 有突出贡献的 contributor。需要独立完成至少一个 feature 或修复重大 bug。
```
Comment (gingerkidney) [inline community-membership-zh_CN.md]:
```suggestion
由 Maintainer 或 PMC 推荐，是对 TiKV 有突出贡献的 Contributor。需要独立完成至少一个 feature 或修复重大 bug。
```

File: community-membership-zh_CN.md [hunk 24]
```diff
@@ -0,0 +1,72 @@
...
+
+### 如何成为
+
+* 表现出良好的技术判断力
+* 至少是两个以上子项目的 reviewer
```
Comment (gingerkidney) [inline community-membership-zh_CN.md]:
```suggestion
* 至少是两个以上子项目的 Reviewer
```

File: community-membership-zh_CN.md [hunk 25]
```diff
@@ -0,0 +1,72 @@
...
+* 审阅或合并了一定数量的 TiKV 代码
+* 对 TiKV 性能提升做出突出贡献；修复了重大 bug
+* 由两名或以上在任 Maintainer 提名通过
+
+### 责任与义务
+
+* 整体把控项目的代码质量 
+* 指导 contributor 和 reviewer 对 TiKV 持续贡献
```
Comment (gingerkidney) [inline community-membership-zh_CN.md]:
```suggestion
* 指导 Contributor 和 Reviewer 对 TiKV 持续贡献
```

File: community-membership-zh_CN.md [hunk 26]
```diff
@@ -0,0 +1,72 @@
...
+* 定期参与 Committer 会议，对项目发展有优先知情权
+
+<h2 id="reviewer">Reviewer</h2>
+
+负责审查社区提交的代码，保证代码的质量和正确性，从 contributor 中产生。
```
Comment (gingerkidney) [inline community-membership-zh_CN.md]:
```suggestion
负责审查社区提交的代码，保证代码的质量和正确性，从 Contributor 中产生。
```

File: community-membership-zh_CN.md [hunk 27]
```diff
@@ -0,0 +1,72 @@
...
+* 定期参加专项兴趣小组（SIG）的研讨会
+* 对代码拥有 LGTM 的权限
+
+## Active Contributor
+
+活跃的社区贡献者。
+
+### 如何成为
+* 一年内对 TiKV 贡献 8 个 pr 及以上
+* 得到两位现任 reviewer  的认可
```
Comment (gingerkidney) [inline community-membership-zh_CN.md]:
```suggestion
* 得到两位现任 Reviewer  的认可
```

File: sig-governace/SIG-GOVERNACE-zh_CN.md [hunk 28]
```diff
@@ -0,0 +1,75 @@
...
+
+参见 [Committer 定义](/community-membership-zh_CN.md#committer)
+
+### Tech Lead
+
+* 数量 2 - 3 名
+* 负责组织 SIG 内部成员的培训，学习
+* 对 SIG 内部产生的 Proposal 需要组织讨论以及作出决策
+* 负责 SIG 的活跃以及产出
+* 需要带领 SIG 产生更多的 Revewer，Commiter
```
Comment (gingerkidney) [inline sig-governace/SIG-GOVERNACE-zh_CN.md]:
```suggestion
* 需要带领 SIG 产生更多的 Revewer，Committer 
```

File: GOVERNACE-zh_CN.md [hunk 29]
```diff
@@ -0,0 +1,51 @@
...
+
+![community organization](media/governace/community_organization.png)
+
+社区架构分成两个部分，第一部分是社区角色架构图，第二部分为社区小组内部的角色分工，以及社区角色的晋升路径。
+
+PMC：项目管理委员会，为项目核心管理团队，参与 roadmap 制定与社区相关的重大决议。
+
+Developer Group：
+
+* Maintainer：项目的规划和设计者，拥有合并主干分支的权限，从 Committer 中产生。
```
Comment (dcalvin) [inline GOVERNACE-zh_CN.md]:
```suggestion
* Maintainer：项目的规划和设计者，拥有合并分支至主干的权限，从 Committer 中产生。
```

File: GOVERNACE-zh_CN.md [hunk 30]
```diff
@@ -0,0 +1,51 @@
...
+* Committer：由 maintainer 或 PMC 推荐，对 TiKV 有突出贡献的 contributor，需要独立完成至少一个 feature 或修复重大 bug。
+* Reviewer：进阶 contributor，负责 review 社区代码，对代码有 LGTM 权限，可以参与到后续项目的设计中。
+* Active Contributor：活跃的社区贡献者，有两位 reviewer 以上的社区角色提名通过。可以参与到 SIG 中，至少合并 8 个 PR 以上。
+* Contributor：社区贡献者，在 TiKV 核心 repo 有一个以上被合并的 PR。
```
Comment (dcalvin) [inline GOVERNACE-zh_CN.md]:
```suggestion
* Contributor：社区贡献者，在 TiKV 名下的 Repo 里有一个以上被合并的 PR。
```

File: GOVERNACE-zh_CN.md [hunk 31]
```diff
@@ -0,0 +1,51 @@
...
+
+* SIG：专项兴趣小组，目前为邀请制。成员为在某一专项活跃的 contributor。小组最终目的为培养社区人才，帮助大家晋升为更高阶的 reviewer，committer 以及 maintainer。
+
+* WG：开放的工作组，社区成员可以无门槛加入。工作组是为了完成一个短期项目而建立的临时小组，项目完成即解散。一个工作组可以横跨多个兴趣小组。经过一段时间的培训后优秀的社区小伙伴会被选入某个兴趣小组进行进一步深造。
+### 内部角色：
+
+* Tech Leader：小组技术负责人，对整个小组的技术走向进行把控。负责制定学习计划、成员考核制度等。一个小组可以有多个 Tech Leader。
+* 其余角色见社区组织架构图
+
+## 晋升通道：
```
Comment (dcalvin) [inline GOVERNACE-zh_CN.md]: 这一部分是不是可以删掉，具体内容其实在上面的部分已经说明。另外  community-membership-zh_CN.md 里也有详细说明。

File: community-membership-zh_CN.md [hunk 32]
```diff
@@ -0,0 +1,72 @@
+# 社区角色
+
+## Maintainer
+
+项目的规划和设计者，拥有合并主干分支的权限，从 Committer 中产生。
```
Comment (dcalvin) [inline community-membership-zh_CN.md]:
```suggestion
项目的规划和设计者，拥有合并分支至主干的权限，从 Committer 中产生。
```

File: community-membership-zh_CN.md [hunk 33]
```diff
@@ -0,0 +1,72 @@
...
+* 及时响应社区提交的 PR
+* 定期参加专项兴趣小组（SIG）的研讨会
+* 对代码拥有 LGTM 的权限
+
+## Active Contributor
+
+活跃的社区贡献者。
+
+### 如何成为
+* 一年内对 TiKV 贡献 8 个 pr 及以上
```
Comment (dcalvin) [inline community-membership-zh_CN.md]:
```suggestion
* 一年内对 TiKV 贡献 8 个 PR 及以上
```

File: community-membership-zh_CN.md [hunk 34]
```diff
@@ -0,0 +1,72 @@
...
+* 得到两位现任 reviewer  的认可
+
+### 责任与义务
+* 自动获得进入专项兴趣小组（SIG）的资格
+
+## Contributor
+
+* 社区贡献者，在 TiKV 核心 repo 有一个或以上被合并的 PR。
```
Comment (dcalvin) [inline community-membership-zh_CN.md]:
```suggestion
* 社区贡献者，在 TiKV 名下 repo 有一个或以上被合并的 PR。
```

File: sig-governace/SIG-GOVERNACE-zh_CN.md [hunk 35]
```diff
@@ -0,0 +1,75 @@
+# SIG Governance
+SIG 全称 Special Interest Group。SIG 在社区架构中的位置请见：[社区架构 & 角色](/GOVERNACE-zh_CN.md#community_organization)。SIG 主要聚集一批 Active Contributor，对某一个或多个 TiKV 的模块深入研究 & 贡献，在 SIG 内部一步步晋升为 Reviewer，Committer。 
+
+## SIG 角色以及组织治理
+
+SIG 遵循 [社区治理规范](/GOVERNACE-zh_CN.md) 中的角色定义。SIG 需要遵守以下规则：
+
+* 在开始一个 SIG 之前，需要提前确立好章程，创建章程请参考：创建 SIG 章程步骤。
```
Comment (dcalvin) [inline sig-governace/SIG-GOVERNACE-zh_CN.md]: > 创建 SIG 章程步骤。

链接？

File: sig-governace/SIG-GOVERNACE-zh_CN.md [hunk 36]
```diff
@@ -0,0 +1,75 @@
...
+* 定期组织会议，至少每双周组织一次，除非法定节假日，并且产出会议记录
```
Comment (dcalvin) [inline sig-governace/SIG-GOVERNACE-zh_CN.md]:
```suggestion
* 定期组织会议，至少每双周组织一次（节假日或其他需推迟或顺延的情况需提前说明），并且产出会议记录
```

File: community-membership-zh_CN.md [hunk 37]
```diff
@@ -0,0 +1,72 @@
...
+<h2 id="reviewer">Reviewer</h2>
+
+负责审查社区提交的代码，保证代码的质量和正确性，从 contributor 中产生。
+
+### 如何成为
+
+* 熟悉代码库
+* 为 TiKV 子模块贡献一定数量的代码
+* 在专项兴趣小组（SIG）中表现优秀
+* 获得至少两位在任 Committer 的提名通过
```
Comment (dcalvin) [inline community-membership-zh_CN.md]: 上述后两条是否可以并为一条。表现优秀单独列出却没有给出具体衡量标准，如果列为一条可理解为，因“表现优秀被提名”。

File: sig-governace/SIG-GOVERNACE-zh_CN.md [hunk 38]
```diff
@@ -0,0 +1,75 @@
...
+* 确立 SIG 可分配任务以及难易程度，由 SIG 成员认领
+
+其他请参见：[SIG 生命周期](#sig_lifetime)
+
+## 角色
+
+<h3 id="active_contributor">Active Contributor</h3>
+
+* 区别于社区的 Active Contributor， 指被推举进入本 SIG 的 Active Contributor。
+* 需要保持 SIG 中至少一个模块的活跃度
```
Comment (dcalvin) [inline sig-governace/SIG-GOVERNACE-zh_CN.md]: 活跃度如何衡量？

File: sig-governace/SIG-GOVERNACE-zh_CN.md [hunk 39]
```diff
@@ -0,0 +1,75 @@
...
+SIG 的成员可以晋升到更高的角色，规则由各 SIG 自行制定。每一次晋升需由更高一级角色或以上角色的两位成员提名。
+
+## 成员退出机制
+
+SIG 的成员在一段时间的不活跃之后，该成员会暂时请出 SIG。退出机制，由各 SIG 自行制定。
+
+## 组织治理
+
+* SIG 需要 bi-weekly 组织一次会议
+* 应由 Tech Lead 发起除非被委任给了其他的成员
```
Comment (dcalvin) [inline sig-governace/SIG-GOVERNACE-zh_CN.md]:
```suggestion
* 由 Tech Lead 或是其他被委任的成员发起
```

File: sig-governace/SIG-GOVERNACE-zh_CN.md [hunk 40]
```diff
@@ -0,0 +1,75 @@
...
+1. 复制 [SIG 章程模板](SIG-CHARTER-zh_CN.md)
+2. 修改模板中需要为新 SIG 定义的内容
+3. 向 [TiKV Community](https://github.com/tikv/community) 发起 Pull Request，将相应的 SIG 章程提交到 [SIGs 目录](/SIGs)
+4. PR 被批准合并之后，会由社区委员会公布新的 SIG
+
+<h2 id="sig_lifetime">SIG 生命周期</h2>
+
+### 创建
+
+1. 确保所有的 SIG Technical Leads，和其他角色至少是 [Active Contributor](#active_contributor)。
```
Comment (dcalvin) [inline sig-governace/SIG-GOVERNACE-zh_CN.md]:
```suggestion
1. 确保所有的 SIG Technical Leads 和其他角色至少是 [Active Contributor](#active_contributor)。
```

File: sig-governace/SIG-GOVERNACE-zh_CN.md [hunk 41]
```diff
@@ -0,0 +1,75 @@
...
+3. 遵从 [创建 SIG 章程步骤](#step_to_create_sig_charter) 创建一个 SIG 的章程
```
Comment (dcalvin) [inline sig-governace/SIG-GOVERNACE-zh_CN.md]:
```suggestion
3. 遵从 [创建 SIG 章程步骤](#step_to_create_sig_charter) 创建 SIG 的章程
```

File: sig-governace/SIG-GOVERNACE-zh_CN.md (hunk 38 above)
Comment (winkyao) [reply sig-governace/SIG-GOVERNACE-zh_CN.md]: 这个可以由各个 SIG 自己定义

Comment (winkyao) [reply sig-governace/SIG-GOVERNACE-zh_CN.md]: 活跃度在各个 SIG 中定义

File: sig-governace/SIG-CHARTER-TEMPLATE-zh_CN.md [hunk 42]
```diff
@@ -0,0 +1,50 @@
...
+### 与 [SIG Governace](/sig-governace/SIG-GOVERNACE-zh_CN.md) 所不同的
+
+* 不相同的列表
+* 阐述不相同的原因以及目的
+* 如果本 SIG 不需要 Tech Leads 角色也在这里阐述
+
+
+
+
+
```
Comment (dcalvin) [inline sig-governace/SIG-CHARTER-TEMPLATE-zh_CN.md]: Remove extra blank lines.

**Comment by winkyao** [comment]
@dcalvin PTAL

**Comment by gingerkidney** [comment]
LGTM

**Pull request merged**